Framework/src/ObjectsManager.cxx

```diff
@@ -30,7 +30,7 @@ namespace core
 
 ObjectsManager::ObjectsManager(TaskConfig& taskConfig) : mTaskName(taskConfig.taskName)
 {
-  startPublishing(&mObjectsList, MonitorObject::SYSTEM_OBJECT_PUBLICATION_LIST);
+//  startPublishing(&mObjectsList, MonitorObject::SYSTEM_OBJECT_PUBLICATION_LIST);
```

Comment:
It should be completely redundant when we send MOs in a vector or list. In this case, it was destroying the merging, hence I commented it out.

Reply:
I think that there is a misunderstanding on what is this mObjectsList. If I am not mistaken it was meant for the GUIs to know what are the objects being published by a certain Task.

Reply:
According to the commit:
_Add support for the publication of the list of all published objects._

_This is needed by the CCDB backend as we have no ways of asking for the list of objects. It introduces the (first) so-called system object, i.e. an object that is transmitted by tasks behind the scene the same way the explicit MonitorObjects are. At the moment, the list of objects is a comma separated list of names. It is interpreted by the CCDB backend to provide the list of published objects. (QC-53)_

So I will try to find another way to make this work with current merger.

Reply:
I added a check in `HistoMerger` to accept classes which are TH1, so that object list string does not interfere.

Reply:
Thank you very much for checking this for me. It saved me quite some effort. 

At the time it was needed, but I am not sure it is still needed. The GUI use an Information Service (the stupid one I wrote or the future official one) to know what objects are being published by a task (https://alice.its.cern.ch/jira/browse/OGUI-147). For the case when we explore all the objects published (in the past) the CCDB now provides an API for it. 

My take is that it can stay commented out.